website/content/blog/2023-10-the-road-to-scaphandre-v1.0-challenges-and-improvements-to-come-on-IT-energy-consumption-evaluation.md

```diff
@@ -0,0 +1,104 @@
+---
+title: The road to Scaphandre v1.0 : Challenges and improvements to come on IT energy consumption evaluation
```

Comment:
```suggestion
title: The road to Scaphandre v1.0 - Challenges and improvements to come on IT energy consumption evaluation
```
We get a YAML error otherwise